app/config/parameters_common.yml

```diff
@@ -19,4 +19,6 @@ parameters:
   container.autowiring.strict_mode: true
   shopsys.entity_extension.map:
     Shopsys\FrameworkBundle\Model\Article\Article: Shopsys\ShopBundle\Model\Article\Article
-    Shopsys\FrameworkBundle\Model\Product\Product: Shopsys\ShopBundle\Model\Product\Product
\ No newline at end of file
+    Shopsys\FrameworkBundle\Model\Product\Product: Shopsys\ShopBundle\Model\Product\Product
+    Shopsys\FrameworkBundle\Model\Category\Category: Shopsys\ShopBundle\Model\Category\Category
+    Shopsys\FrameworkBundle\Model\Category\CategoryDomain: Shopsys\ShopBundle\Model\Category\CategoryDomain
```

Comment:
sorry but this has got problem with blank line on the end

Reply:
Done